### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by ZzzjzzZ** [NanqingD/DeepLabV3-Tensorflow#20]
你好，
     非常感谢你的工程，但我在训练过程中遇到一个问题。 `seg_logits = tf.boolean_mask(raw_output_up, mask)`  raw_output_up 是（1,321,321，21）， mask 是（321， 321）， 报错提示 `Shapes (1, 321) and (321, 321) are incompatible` 请问这应该怎么解决呢？
